source/-XML/XMLWriter.cpp

```diff
@@ -586,12 +587,12 @@ template<class T>
 void
 XMLWriter::
 writeNodes(
-    AnyList<T> const& list)
+    doc::List<T> const& list)
 {
     if(list.empty())
         return;
-    for(auto const& node : list)
-        writeNode(node);
+    for(const auto& node : list)
+        writeNode(*node);
```

Comment:
you changed east const to west const.... I am going to mandate east const in this project